geomesa-quickstart-kafka/pom.xml

```diff
@@ -28,10 +33,6 @@
             <groupId>org.apache.zookeeper</groupId>
             <artifactId>zookeeper</artifactId>
         </dependency>
-        <dependency>
-            <groupId>commons-cli</groupId>
-            <artifactId>commons-cli</artifactId>
-        </dependency>
         <dependency>
             <groupId>log4j</groupId>
             <artifactId>log4j</artifactId>
```

Comment:
Should we nuke the `<properties>` tag here like other places?

Reply:
which properties? i don't see any in this file...

Reply:
It's in the hidden, unedited section. `<profile><properties>`

Reply:
ah thanks, yeah i did nuke it in the next PR when i updated the prop names